releases/v20.2.0-alpha.2.md

```diff
@@ -0,0 +1,350 @@
+---
+title: What's New in v20.2.0-alpha.2
+toc: true
+summary: Additions and changes in CockroachDB version v20.2.0-alpha.2
+---
+
+## July 22, 2020
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-alpha.2.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-alpha.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-alpha.2.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-alpha.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.2.0-alpha.2
+~~~
+
+### Backward-incompatible changes
+
+- The [`SHOW RANGE FOR ROW`](../v20.2/show-range-for-row.html) now takes a tuple of the row's index columns instead of the full column set of the row. [#50479][#50479]
+- Previously, issuing a `SIGTERM` signal twice or after another signal initiated a hard shutdown for a node. Now the first `SIGTERM` signal initiates a graceful shutdown and further occurrences of `SIGTERM` are ignored. To initiate a hard shutdown, issue `SIGINT` two times (or issue a `SIGINT` signal once after a `SIGTERM` signal). [#50539][#50539]
+- Specifying the same option multiple times in the `WITH` clause of the [`BACKUP`](../v20.2/backup.html) statement now results in an error message. Additionally, quoted option names are not allowed anymore. [#50723][#50723]
+
+### Security updates
+
+- HTTP endpoints beginning with `/debug/` now require a valid [admin](../v20.2/authorization.html) login session. [#50483][#50483]
+
+### General changes
+
+- Go 1.14 is now the minimum required version necessary to build CockroachDB. [#50671][#50671]
+- The docker image available with CockroachDB now includes the GEOS files required to work with Geospatial types in CockroachDB. [#50362][#50362]
```

Comment:
Mind making "Geospatial types" a link to https://www.cockroachlabs.com/docs/v20.2/spatial-data.html ?

Reply:
Done.